screens/auth/SignUpScreen.js

```diff
@@ -88,6 +89,7 @@ export default class SignUpScreen extends React.Component {
   // Sign up function. It sets the user token in local storage
   // to be the fname + lname and then navigates to homescreen.
   _asyncSignUp = async customerId => {
+    await Analytics.setUserId(customerId);
```

Comment:
same suggestion from login, to move setting the customerId above where Sentry registers the customer on a successful signup above line 140
```
   // if signup works, register the user
   await Analytics.setUserId(customer.id);
      Sentry.configureScope(scope => {
```